Incorrect setcap path on Alpine Linux

**In raising this issue, I confirm the following (please check boxes, eg [X]) Failure to fill the template will close your issue:**

- [x] I have read and understood the [contributors guide](https://github.com/pi-hole/pi-hole/blob/master/CONTRIBUTING.md).
- [x] The issue I am reporting can be *replicated*
- [x] The issue I am reporting isn't a duplicate 

**How familiar are you with the codebase?:**

_1_

---

**[BUG | ISSUE] Expected Behaviour:**

`make install` completes successfully.

**[BUG | ISSUE] Actual Behaviour:**

`make install` fails with command not found.

**[BUG | ISSUE] Steps to reproduce:**

- `make`
- `make install`

**Log file output [if available]**

```
install -m 0755 pihole-FTL /usr/bin
/sbin/setcap CAP_NET_BIND_SERVICE,CAP_NET_RAW,CAP_NET_ADMIN+eip /usr/bin/pihole-FTL
make: /sbin/setcap: Command not found
make: *** [Makefile:127: install] Error 127
```

**Device specifics**

Hardware Type: PC
OS: Alpine Linux


_This template was created based on the work of [`udemy-dl`](https://github.com/nishad/udemy-dl/blob/master/LICENSE)._

--- a/Makefile
+++ b/Makefile
@@ -152,6 +152,7 @@ version.h: version~
 	@echo "Making FTL version on branch $(GIT_BRANCH) - $(GIT_VERSION) ($(GIT_DATE))"
 
 prefix=/usr
+SETCAP = $(shell which setcap)
 
 # install target just installs the executable
 # other requirements (correct ownership of files, etc.) is managed by
@@ -159,4 +160,4 @@ prefix=/usr
 install: pihole-FTL
 	mkdir -p $(DESTDIR)$(prefix)/bin
 	install -m 0755 pihole-FTL $(DESTDIR)$(prefix)/bin
-	/sbin/setcap CAP_NET_BIND_SERVICE,CAP_NET_RAW,CAP_NET_ADMIN+eip $(DESTDIR)$(prefix)/bin/pihole-FTL
+	$(SETCAP) CAP_NET_BIND_SERVICE,CAP_NET_RAW,CAP_NET_ADMIN+eip $(DESTDIR)$(prefix)/bin/pihole-FTL

--- a/Makefile
+++ b/Makefile
@@ -151,13 +151,13 @@ version.h: version~
 	@echo '#endif // VERSION_H' >> "$@"
 	@echo "Making FTL version on branch $(GIT_BRANCH) - $(GIT_VERSION) ($(GIT_DATE))"
 
-prefix=/usr
+PREFIX=/usr
 SETCAP = $(shell which setcap)
 
 # install target just installs the executable
 # other requirements (correct ownership of files, etc.) is managed by
 # the service script on sudo service pihole-FTL (re)start
 install: pihole-FTL
-	mkdir -p $(DESTDIR)$(prefix)/bin
-	install -m 0755 pihole-FTL $(DESTDIR)$(prefix)/bin
-	$(SETCAP) CAP_NET_BIND_SERVICE,CAP_NET_RAW,CAP_NET_ADMIN+eip $(DESTDIR)$(prefix)/bin/pihole-FTL
+	mkdir -p $(DESTDIR)$(PREFIX)/bin
+	install -m 0755 pihole-FTL $(DESTDIR)$(PREFIX)/bin
+	$(SETCAP) CAP_NET_BIND_SERVICE,CAP_NET_RAW,CAP_NET_ADMIN+eip $(DESTDIR)$(PREFIX)/bin/pihole-FTL

